README.md

```diff
@@ -105,6 +105,15 @@ services:
 
 ## Running on Linux
 
+### Arch Linux
+
+```shell
+paru -Sy backrest
+sudo systemctl enable --now backreset@$USER.service
```

Comment:
nit: `backrest`?

same question as above, where does the service definition come from?